frontend/packages/design-system/src/themes/Button.ts

```diff
@@ -1,12 +1,19 @@
+import textTheme from './Text';
+
 export default {
+  defaultProps: {
+    _text: {
+      ...textTheme.variants.button,
```

Comment:
I thought we didn't want to use `defaultProps` for styles?

Reply:
I forget exactly, but I feel like props are the wrong place for styles as they are easy for the user to override. In other parts of the theme you've used `baseStyle` for styles shared across themes. Maybe that's a better place for this.